frontend/src/store/modules/visitors/actions.js

```diff
@@ -151,4 +155,12 @@ export default {
                 return Promise.reject(response);
             });
     },
+    [FETCH_EACH_BUTTON_DATA]: (context, buttonTypes) => {
+        buttonTypes.each((buttonType) => {
+            let data = {
+                button: buttonType,
+            };
+            context.dispatch([FETCH_BUTTON_DATA], data);
```

Comment:
it won't work:
context.dispatch([FETCH_BUTTON_DATA], data);

what is?
[FETCH_BUTTON_DATA]